<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastrarPalavras</class>
 <widget class="QMainWindow" name="cadastrarPalavras">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CADASTRAR NOVAS PALAVRAS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>favicon.ico</normaloff>favicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(154, 154, 154);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>300</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>245</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="inputPalavra">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>280</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid black;
border-radius: 8px;
padding: 2px;
color: #000;
background-color: rgb(200, 200, 200); 
</string>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Nova Palavra Secreta</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="inputDica">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>461</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid black;
border-radius: 8px;
padding: 2px;
color: #000;
background-color: rgb(200, 200, 200); 
</string>
       </property>
       <property name="placeholderText">
        <string>Dica da palavra secreta:</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>40</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btnCadastrar">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>250</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 2px solid;
	border-radius: 8px;
	color: #878787;
	background-color: rgb(200, 200, 200); 
}

QPushButton:hover {
	color: rgb(200, 200, 200);
	background-color: #878787;
}

QPushButton:pressed {
	color: rgb(200, 200, 200);
	background-color: black;
}</string>
       </property>
       <property name="text">
        <string>Cadastrar Palavra Secreta</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSair">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>5</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 2px solid;
	border-radius: 8px;
	color: #878787;
	background-color: rgb(200, 200, 200); 
}

QPushButton:hover {
	color: rgb(200, 200, 200);
	background-color: #878787;
}

QPushButton:pressed {
	color: rgb(200, 200, 200);
	background-color: black;
}</string>
       </property>
       <property name="text">
        <string>Sair</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icones.qrc"/>
 </resources>
 <connections/>
</ui>
